stories/25-Modal.stories.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import { MLModal, MLButton } from '@marklogic/design-system'
+import { withKnobs, boolean } from '@storybook/addon-knobs'
+import {
+  DraggableModalProvider,
+} from 'ant-design-draggable-modal'
+import { action } from '@storybook/addon-actions'
+
+export default {
+  title: 'Feedback/MLModal',
+  component: MLModal,
+  decorators: [withKnobs],
+}
+
+function useVisible() {
+  const [visible, setVisible] = useState(false)
+  const toggle = (e) => {
+    setVisible(!visible)
+    return action('toggle modal')(e)
+  }
+  const show = (e) => {
+    setVisible(true)
+    return action('show modal')(e)
+  }
+  const hide = (e) => {
+    setVisible(false)
+    return action('hide modal')(e)
```

Comment:
Actions should be named after the event, not a description. Use `onOk` and `onCancel` for the action names.

Reply:
Can't do that in this case and also reuse the `hide` function since it pertains to both `onOk` and `onCancel`. I think having a common handler is a good way to do it in production which makes it a valuable example in the story. I'm not super attached though, so I'll change it if you want. Not sure how to make it work with my `useVisible` function though.